composer.json

```diff
@@ -74,7 +74,8 @@
         },
         "optimize-autoloader": true,
         "sort-packages": true,
-        "platform-check": true
+        "platform-check": true,
+        "autoloader-suffix": "PsalmAutoloader"
```

Comment:
This is not required if you use Composer >= 2.6.4, see relevant PR @ https://github.com/composer/composer/pull/11663

Reply:
This is not relevant for us, as the autoloader suffix is reused only if composer.lock is commited, which I've decided not do for psalm itself (only box) for a number of reasons, like https://github.com/vimeo/psalm/issues/10446#issuecomment-1896650439